disnake/interactions/base.py

```diff
@@ -467,6 +468,16 @@ async def edit_original_response(
 
             .. versionadded:: 2.9
 
+        delete_after: Optional[:class:`float`]
+            If provided, the number of seconds to wait in the background
+            before deleting the message we just edited. If the deletion fails,
+            then it is silently ignored.
+
+            Can be up to 15 minutes after the interaction was created
+            (see also :attr:`Interaction.expires_at`/:attr:`~Interaction.is_expired`).
```

Comment:
It would be nice if this was included in `delete_original_response`

Reply:
https://github.com/DisnakeDev/disnake/pull/1014/commits/2f5b8cdf146c4518f9c6835b6a839178dd1425e4